src/wasm/wasm-ir-builder.cpp

```diff
@@ -219,18 +219,15 @@ Result<> IRBuilder::visitExpression(Expression* curr) {
 #define DELEGATE_FIELD_CHILD_VECTOR(id, field)                                 \
   WASM_UNREACHABLE("should have called visit" #id " because " #id              \
                    " has child vector " #field);
-#define DELEGATE_FIELD_SCOPE_NAME_USE(id, field)                               \
-  WASM_UNREACHABLE("should have called visit" #id " because " #id              \
-                   " has scope name use " #field);
-#define DELEGATE_FIELD_SCOPE_NAME_USE_VECTOR(id, field)                        \
-  WASM_UNREACHABLE("should have called visit" #id " because " #id              \
-                   " has scope name use vector " #field);
 
 #define DELEGATE_FIELD_INT(id, field)
 #define DELEGATE_FIELD_INT_ARRAY(id, field)
 #define DELEGATE_FIELD_LITERAL(id, field)
 #define DELEGATE_FIELD_NAME(id, field)
 #define DELEGATE_FIELD_NAME_VECTOR(id, field)
+#define DELEGATE_FIELD_SCOPE_NAME_USE(id, field)
+#define DELEGATE_FIELD_SCOPE_NAME_USE_VECTOR(id, field)
```

Comment:
What does the vector version do that's different?

Reply:
`br_table` is the only instruction that takes a list of labels, so it is the only one that currently uses `NAME_USE_VECTOR`.

I'm removing the error when visiting `NAME_USE` because it turns out the default `visitExpression` implementation Just Works for `BrOn`, even though it includes `NAME_USE`. I'm making the same change for `NAME_USE_VECTOR` for consistency. There's nothing about having a `NAME_USE_VECTOR` that fundamentally means that this default implementation won't work.